test/test_autoextract.py

```diff
@@ -0,0 +1,110 @@
+import os
+import sys
+import pytest
+
+from w3lib.http import basic_auth_header
+from scrapy.http import Request, Response
+from scrapy.spiders import Spider
+from scrapy.utils.test import get_crawler
+
+sys.path.append(os.getcwd())
```

Comment:
why is this line needed? Tests should run for installed scrapy_autoextract. Also, os.getcwd() looks incorrect, because you may invoke these tests while the current folder is arbitrary; cwd is nott necessarily related to path to this file.